connectors-e2e-test/connectors-e2e-test-base/src/main/java/io/camunda/connector/e2e/ZeebeTest.java

```diff
@@ -67,7 +67,7 @@ public ZeebeTest createInstance() {
 
   public ZeebeTest waitForProcessCompletion() {
     ZeebeTestThreadSupport.waitForProcessInstanceCompleted(
-        processInstanceEvent, Duration.of(10, ChronoUnit.SECONDS));
+        processInstanceEvent, Duration.of(60, ChronoUnit.SECONDS));
```

Comment:
What about `20`? Should be enough for most of the tests assuming that containers are started before the tests.